File: WebClient/Data/IOrderItemsService.cs
using System.Collections.Generic;
using System.Threading.Tasks;
using WebClient.Models;

namespace WebClient.Data
{
    public interface IOrderItemsService
    {
        Task CreateOrderItem(int quantity, Menu menu, Order order);
        Task<List<OrderItem>> GetOrderItems(long orderId);

        Task DeleteOrderItem(long orderItemId);
    }
}

File: RestaurantServer/src/main/java/dk/restaurant/services/IOrderItemsService.java
package dk.restaurant.services;

import dk.restaurant.models.OrderItem;

import java.util.List;

public interface IOrderItemsService
{
  OrderItem createOrderItem(OrderItem orderItem);
  List<OrderItem> getOrderItems(long orderId);
  void deleteOrderItem(OrderItem orderItem);
}
